activities/views/posts.py

```diff
@@ -207,8 +208,9 @@ def get_initial(self):
         ] = self.request.identity.config_identity.default_post_visibility
         if self.reply_to:
             initial["reply_to"] = self.reply_to.pk
-            initial["visibility"] = Post.Visibilities.unlisted
```

Comment:
Should reply_to visibility ignore `default_post_visibility` and force `unlisted` when dealing with federating posts?

Reply:
It should actually default to the visibility of the post you're replying to if that's unlisted/followers only/mentions only, or unlisted if that's public/local (we don't want to default it to unlisted if the original message is mentions-only!).